src/libutil/experimental-features.cc

```diff
@@ -163,6 +163,8 @@ constexpr std::array<ExperimentalFeatureDetails, 14> xpFeatureDetails = {{
         .tag = Xp::ReplFlake,
         .name = "repl-flake",
         .description = R"(
+            *Enabled with [`flakes`](#xp-feature-flakes) since 2.19*
```

Comment:
Since apparently we can't just ditch the experimental feature as that would make noise in existing configs, can we at least hide it from view then?

Unfortunately that will require another field in that struct...

Reply:
Then people won't know what it meant.

Reply:
:man_facepalming: So we can't even remove experimental features now? Would it make sense to add some other indirection, such as issuing warnings on stabilised experimental features but not displaying them in the manual?

Reply:
🤦‍♂️ It is not a problem.